web/packages/teleterm/src/mainProcess/mainProcessClient.ts

```diff
@@ -68,5 +68,10 @@ export default function createMainProcessClient(): MainProcessClient {
         cleanup: () => ipcRenderer.removeListener(channel, onThemeChange),
       };
     },
+    downloadAgent() {
+      return ipcRenderer.invoke(
+        'main-process-connect-my-computer-download-agent'
+      );
```

Comment:
When the download fails, the UI is going to show the following:

> Error invoking remote method 'main-process-connect-my-computer-download-agent': Error: Download failed. Requested file may not exist or is temporarily unavailable.

Ideally, we'd like to at least show only this part:

> Download failed. Requested file may not exist or is temporarily unavailable.

Is there any way we can do this without hardcoding a match on the string which could break between Electron updates?

Reply:
There is no way to disable this https://github.com/electron/electron/blob/f959fb0c963701c40f309eb8256530f788d5851e/lib/renderer/ipc-renderer-internal.ts#L20

I agree that is not beautiful but OTOH it's not a big deal to me. We could prepend some string to error messages in `ipcMain` handlers, then split the messages in `ipcRenderer` and remove the first part. But probably it is not worth the effort :(